app/src/main/res/layout/number_validation.xml

```diff
@@ -48,17 +48,25 @@
                 android:hint="@string/hint_validation_name"
                 android:inputType="textPersonName" />
 
-            <Spinner android:id="@+id/phone_cc"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:spinnerMode="dialog"/>
+            <LinearLayout
+                xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please remove xmlns attributes from child tags. Those should belong only to the root tag.

Reply:
will do so